engine/src/main/java/org/destinationsol/game/SolGame.java

```diff
@@ -48,8 +50,6 @@
 import org.destinationsol.game.ship.ShipBuilder;
 import org.destinationsol.game.ship.SloMo;
 import org.destinationsol.game.ship.hulls.HullConfig;
-import org.destinationsol.assets.audio.OggSoundManager;
-import org.destinationsol.assets.audio.SpecialSounds;
```

Comment:
The changes in this file seen irrelevant to those in the rest of the PR and should probably be removed